src/app/functions/server/dashboard.ts

```diff
@@ -142,102 +142,95 @@ export function dashboardSummary(
-          }
-        }
+    // count password
+    if (dataClasses.includes(BreachDataTypes.Passwords)) {
+      summary.totalExposures += increment;
+      summary.allExposures.passwords += increment;
+      if (b.isResolved) {
+        summary.fixedExposures.passwords += increment;
+        summary.dataBreachFixedNum += increment;
+      }
+    }
 
-        // count ssn
-        if (dataClasses.includes(BreachDataTypes.SSN)) {
-          summary.totalExposures++;
-          summary.allExposures.socialSecurityNumbers++;
-          if (b.IsResolved) {
-            summary.fixedExposures.socialSecurityNumbers++;
-            summary.dataBreachFixedNum++;
-          }
-        }
+    // count ssn
+    if (dataClasses.includes(BreachDataTypes.SSN)) {
+      summary.totalExposures += increment;
+      summary.allExposures.socialSecurityNumbers += increment;
+      if (b.isResolved) {
+        summary.fixedExposures.socialSecurityNumbers += increment;
+        summary.dataBreachFixedNum += increment;
+      }
+    }
 
-        // count IP
-        if (dataClasses.includes(BreachDataTypes.IP)) {
-          summary.totalExposures++;
-          summary.allExposures.ipAddresses++;
-          if (b.IsResolved) {
-            summary.fixedExposures.ipAddresses++;
-            summary.dataBreachFixedNum++;
-          }
-        }
+    // count IP
+    if (dataClasses.includes(BreachDataTypes.IP)) {
+      summary.totalExposures += increment;
+      summary.allExposures.ipAddresses += increment;
+      if (b.isResolved) {
+        summary.fixedExposures.ipAddresses += increment;
+        summary.dataBreachFixedNum += increment;
+      }
+    }
 
-        // count credit card
-        if (dataClasses.includes(BreachDataTypes.CreditCard)) {
-          summary.totalExposures++;
-          summary.allExposures.creditCardNumbers++;
-          if (b.IsResolved) {
-            summary.fixedExposures.creditCardNumbers++;
-            summary.dataBreachFixedNum++;
-          }
-        }
+    // count credit card
+    if (dataClasses.includes(BreachDataTypes.CreditCard)) {
+      summary.totalExposures += increment;
+      summary.allExposures.creditCardNumbers += increment;
+      if (b.isResolved) {
+        summary.fixedExposures.creditCardNumbers += increment;
+        summary.dataBreachFixedNum += increment;
+      }
+    }
 
-        // count pin numbers
-        if (dataClasses.includes(BreachDataTypes.PIN)) {
-          summary.totalExposures++;
-          summary.allExposures.pinNumbers++;
-          if (b.IsResolved) {
-            summary.fixedExposures.pinNumbers++;
-            summary.dataBreachFixedNum++;
-          }
-        }
+    // count pin numbers
+    if (dataClasses.includes(BreachDataTypes.PIN)) {
+      summary.totalExposures += increment;
+      summary.allExposures.pinNumbers += increment;
+      if (b.isResolved) {
+        summary.fixedExposures.pinNumbers += increment;
+        summary.dataBreachFixedNum += increment;
+      }
```

Comment:
Should these be "pinNumbers" or just "pins"?

Reply:
Or `personalIdentificationNumbers`, although I'm not thrilled about the `lI` in there. None of these sounds great actually, I don't have a strong opinion.

Reply:
I changed it to `pins`!